### ISSUE TITLE
[Experiment] Change value_cost_weight

### ISSUE THREAD

**Issue by sethtroisi** [tensorflow/minigo#350]
Recently I added a flag `value_cost_weight`
It acts similar to `l2_strength` to scale the contribution of `value_cost` to `combined_cost`

Remember that `combined_cost = policy_cost + value_cost + l2_cost` [[1]](https://github.com/tensorflow/minigo/blob/335431305897aa42726029c9974100b7dc2bf575/dual_net.py#L221) and

`value_cost = FLAGS.value_cost_weight * tf.reduce_mean(value_err ** 2)`
`l2_cost = FLAGS.l2_strength * tf.add_n([tf.nn.l2_loss(v) for v in reg_vars])`

Today I trained several networks while varying `value_cost_weight`.
I selected golden_chunks 250, 251, 252, and 253 to train with (this is 4 x 2 million positions which we used to trained fairly strong models with see [CloudyGo](http://cloudygo.com/v9-19x19/eval-graphs) for model strength)

I choose 10k steps (2.5 epochs) because that seems to be about the right number for a model to have converged most of the way (I would do 16k = 4 epochs if I did this again).

Results:
![image](https://user-images.githubusercontent.com/10172976/43352578-8208b26c-91da-11e8-929d-dd8c54fcf523.png)
The default setting, `value_cost_weight = 1` over 4 runs averaged `value_cost = 0.82` and `policy_accuracy_top_1 = 0.32` this was fairly stable in 3 of 4 runs (1 run converged ~20% faster)

![image](https://user-images.githubusercontent.com/10172976/43352608-d526c72c-91da-11e8-8ef0-cadec5c71d71.png)
Here I've selected the runs with `value_cost_weight <= 1`
I think of these as prioritizing training the policy head over the value head, and we see that policy converges faster but not to a higher accuracy. Also if `value_cost_weight` is too low (maybe <=0.2) it appears that the value head doesn't learn simultaneously which is bad.

![image](https://user-images.githubusercontent.com/10172976/43352642-5821076e-91db-11e8-9603-6afebbc3f1f4.png)
Here I've selected the runs with `value_cost_weight >= 1`
I think of these as prioritizing training the value head over the policy head. Unlike with our previous set these *don't* seem to converges faster or to a higher accuracy. Slightly surprising if `value_cost_weight` is too large (10 in this case) value head appears to learn nothing for 7k steps then it makes a sudden improvement but ends up worse in both policy accuracy and value error. I think this goes to show that the value head and policy head are both learning similar things in the shared trunk and by turning one of the heads off we reduce the data the shared trunk can train on.

**Comment by sethtroisi** [comment]
I failed to normalize cost after changing `value_cost_weight` which if I understand correctly implies I increased learning rate slight.

with `value_cost_weight = 2`, `policy_cost ~= 4`, and `l2_cost ~= 0.2
`cost = 2 * 0.8 + ~4 + ~0.2 = 5.8`
compared with original `cost = 0.8 + ~4 + ~0.2 = 4.8` which corresponds to a ~20% increase in LR which hopefully wouldn't be a big impact
